tests/integration.rs

```diff
@@ -885,16 +885,24 @@ mod integration_tests {
         ) -> Result<SpinTestController> {
             // start Spin using the given application manifest and wait for the HTTP server to be available.
             let url = format!("127.0.0.1:{}", get_random_port()?);
-            let mut args = vec!["up", "--file", manifest_path, "--listen", &url];
-            args.extend(spin_args);
-            if let Some(b) = bindle_url {
-                args.push("--bindle-server");
-                args.push(b);
-            }
-            for v in spin_app_env {
-                args.push("--env");
-                args.push(v);
-            }
+            let env_args = spin_app_env.iter().flat_map(|e| ["--env", e]).collect();
+
+            let trigger_args = vec!["--file", manifest_path, "--listen", &url];
+
+            let bindle_args = match bindle_url {
+                Some(url) => vec!["--bindle-server", url],
+                None => vec![],
+            };
+
+            let args = vec![
+                vec!["up"],
+                spin_args.to_vec(),
+                bindle_args,
+                env_args,
+                trigger_args,
+            ]
+            .into_iter()
+            .flatten();
```

Comment:
Probably worth mentioning I had to change this test. Clap changed the way it parses, so that --env was being seen as part of spin trigger and not spin up.

I could try to force it, but it's not going to be pretty. The old behavior seemed less correct since --env appeared after other args which were going to spin trigger.